May 2025 bugfixes

ilkober opened the pull request.

BiccyThiccy committed:
Some idea and leader descriptions, 1 typo fix

BiccyThiccy committed:
Slightly more accurate bookmark loc

BiccyThiccy committed:
Francisco Rolao Preto description

BiccyThiccy committed:
Finishing up the descriptions

BiccyThiccy committed:
Fixing 1 live error

Since this guy already exists and has this trait, his given leader role needs to be effect-tooltipped

BiccyThiccy committed:
Fix NOR arresting Trotsky *then* deporting sending MEX the wrong event

BiccyThiccy committed:
ITA - Carrying over one vanilla AAT change, removing debugger(?) unneeded tt

BiccyThiccy committed:
SIA bag of fixes

Improves some incorrectly capitalized loc, optimizes some checks, and fixes puppet Indochina getting an event asking for land in Laos/Cambodia

BiccyThiccy committed:
Merge branch 'Beta-Edits' into May-2025-Bugfix

ilkober committed:
Little code cleanup

ilkober committed:
Removing old pre-character instances

marijn211 committed:
Hist fixes+No alt-hist Dutch election on hist

BiccyThiccy committed:
Giving authoritarian SAO its demo flag (it has none in live)

BiccyThiccy committed:
Fixes from Marijn and I

BiccyThiccy committed:
Description for Dom Duarte Nuno

marijn211 committed:
Ported 2 fixes from the open beta, ignore any errors on the non-beta

marijn211 committed:
Fix Belgium DLC tree having an unselectable naval theorist

Commit:
Otto fix

Bugfix: [GD] After completing focus "Invite Otto to the Thorne" as Hungary you will get Otto von Habsburg as a field marshal

marijn211 committed:
Update soviet.txt

marijn211 committed:
Fix events in Scotland tree

marijn211 committed:
Update r56_argentina.txt

ilkober committed:
Replacing some fieldmarshal generators

Commit:
Swiss forts fix

[RT56] Swiss fortification branch now gives the correct amount of forts

BiccyThiccy committed:
Fixing two vanilla Portugal error-spammers

- One is an uncommented advisor that Spec removed
- One is removing some has_wargoal_against scoping that is causing error log spam here AND in vanilla

Commit:
Replaced a really bad nat spirit icon for Switzerland

BiccyThiccy committed:
PER - Implementing some War Effort Changes

marijn211 committed:
Update r56_switzerland.txt

marijn211 commented on common/national_focus/persia.txt:
```diff
@@ -5068,11 +5077,11 @@ focus_tree = {
-			has_war_with = ENG
-			has_war_with = SOV
-			has_global_flag = SOV_agreed_anglo_soviet_invasion_of_iran_flag
+			#r56: simplified and restricted the focus since the vanilla version of this focus is defective		- Commenting this section because PDX reworked the effects, if someone wants to restore go ahead.
```
My version of this focus has to be restored, vanilla's version is still completely broken @BiccyThiccy

BiccyThiccy replied:
Sorry, I'll restore how it was shortly.

marijn211 reviewed (dismissed):
Above

marijn211 committed:
Fix missing French claims

BiccyThiccy committed:
Revert import of PER_swear_fealty changes + fixing colliding focus

The two focus's that appear after Operation Countenance were colliding with the yalta focus's, the other change is just moving the yalta branch down one more

Commit:
improved bop gfx organization

Commit:
Update r56_bulgaria.txt

Commit:
Belgium fixes

BiccyThiccy committed:
Fixing (I think) every instance where Spain can re-add Recovering From the Spanish Civil War

BiccyThiccy committed:
Various Spelling Fixes

BiccyThiccy committed:
Carlism idea will no longer be stuck on Falangist Spain if you win too soon

BiccyThiccy committed:
Importing Apr16 Beta Hotfix Changes to two PER focus's

marijn211 committed:
Event

marijn211 committed:
Honduras bugfix

marijn211 committed:
Improve old Soviet historical AI

BiccyThiccy committed:
Changing some duplicate idea icons

BiccyThiccy committed:
Spelling fixes, mostly for FRA

marijn211 committed:
Fix improperly commented code

marijn211 committed:
Remove incorrect Dutch operation names

marijn211 committed:
Update LAT - Latvia.txt

marijn211 committed:
Typo fix

marijn211 committed:
Fix Smurf

marijn211 committed:
Multiple bugfixes

marijn211 committed:
Fix typo previous commit

marijn211 committed:
Fix missing file extension

marijn211 committed:
Historical&Old Soviet Tree polish, see desc

- HUN doesnt demand Transylvania on historical, instead letting Germany do their thing.
- Cleaned up old Soviet AI strategies a bit
- Allow old Soviet tree to assassinate Trotsky
- Fix old Soviet focus "Claim on Sinkiang" not allowing volunteers

marijn211 committed:
Trotsky in Rt56 tree to character form

marijn211 committed:
POL divided if Axis defeats Allies before Barb.

marijn211 committed:
Improved the positioning of a few cities/ports

marijn211 committed:
Fix broken icon

marijn211 reviewed (approved):
Clean error log, good diff, up-to-date with main

marijn211 committed:
Further improved old Sov historical AI

BiccyThiccy committed:
Triumph of Synarchism will no longer float

BiccyThiccy committed:
Fixing 1 awkward-looking offset

marijn211 commented:
Marking ready for tomorrow but please dont merge, Ill do that myself

marijn211 committed:
Merge branch 'Beta-Edits' into may-2025-bugfixes
